server_addon/max/client/ayon_max/api/lib.py

```diff
@@ -284,6 +282,16 @@ def reset_frame_range(fps: bool = True):
         frame_range["frameStartHandle"], frame_range["frameEndHandle"])
 
 
+def set_fps():
```

Comment:
I'm personally quite confused that a function `set_fps` does not take arguments. `set_timeline` does, `set_render_frame_range` does - why not here?

In maya for example the equivalent to this in AYON is:
```python
set_scene_fps(get_fps_for_current_context())
```
See [here](https://github.com/ynput/ayon-core/blob/a3aed0d4b7db251fd1db56f3d89e1eaa87200a76/server_addon/maya/client/ayon_maya/api/lib.py#L2533).